fhir/com.b2international.snowowl.fhir.rest/src/com/b2international/snowowl/fhir/rest/FhirCodeSystemController.java

```diff
@@ -15,14 +15,38 @@
  */
 package com.b2international.snowowl.fhir.rest;
 
+import java.util.Collection;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
 import org.springdoc.api.annotations.ParameterObject;
-import org.springframework.web.bind.annotation.*;
```

Comment:
Configure your IDE to automatically apply a wildcard import for both static and non-static import statements when it reaches 5 or more import statements from the same package. Thank you!